test/e2e/go.mod

```diff
@@ -12,13 +12,13 @@ require (
 	github.com/vmware-tanzu/sonobuoy v0.18.2
 	golang.org/x/sync v0.0.0-20200317015054-43a5402ce75a
 	golang.org/x/time v0.0.0-20200416051211-89c76fbcd5d1 // indirect
-	k8s.io/api v0.17.8
-	k8s.io/apimachinery v0.17.8
+	k8s.io/api v0.18.6
```

Comment:
Probably can ditch any changes to test/e2e?

Reply:
I think `make verify` will fail without this. I'll test...

Reply:
you're right

Reply:
change removed - though `make modules` will put it back when run, FYI.